Inc/Utils/control_util.h

```diff
@@ -29,8 +29,8 @@ typedef struct {
 	float timeDiff;
 }PIDvariables;
 
-
-float PID(float err, PIDvariables K){
+//PID control, static to not have multiple implementation error
+static float PID(float err, PIDvariables K){
```

Comment:
change PIDvariables to a pointer. That way it will not go out of scope.
I also see no reason to have this function as a static, beacuse everyone will use this file. It might be nicer to have it as an inline function

Reply:
Problem is that control utils is include by both vel_control as well as geneva, so two declarations exist as it is included twice. You can leave it as static or replace it with inline, both work